nuxt.config.ts

```diff
@@ -16,9 +16,41 @@ export default {
         name: 'description',
         content:
           'We at Club of Programmers Software Development Group are a bunch of designers and developers who aim to encourage the development of technology and innovation in IIT BHU campus (and beyond) by learning, sharing knowledge and solving problems.'
+      },
+      { hid: 'og:type', property: 'og:type', content: 'website' },
+      {
+        hid: 'og:title',
+        property: 'og:title',
+        content: 'COPS SDG'
+      },
+      {
+        hid: 'og:description',
+        property: 'og:description',
+        content: 'We at Club of Programmers Software Development Group are a bunch of designers and developers who aim to encourage the development of technology and innovation in IIT BHU campus (and beyond) by learning, sharing knowledge and solving problems.'
+      },
+      {
+        hid: 'og:image',
+        property: 'og:image',
+        content: 'static/COPS-Logo.webp'
```

Comment:
this won't work instead just post the image on some discord server and copy its link here